netutils/lib_mapper.py

```diff
@@ -124,6 +124,22 @@
     "windows": "windows",
 }
 
+# DNA Center | Normalized
+DNA_CENTER_LIB_MAPPER = {
+    "IOS": "cisco_ios",
+    "IOS-XE": "cisco_ios",
+    "NX-OS": "cisco_nxos",
+    "IOS-XR": "cisco_xr",
+}
+
+# DNA Center | Normalized
```

Comment:
Can you flip this?

Reply:
Oops, good catch! I meant to flip that. I've fixed it and remove the cisco_xe key.